Applications/WebPortal/WebApp/library/source/pages/account_directory_page/group_card.tsx

```diff
@@ -16,8 +16,15 @@ interface Properties {
 
   /** The current filter used on the accounts. */
   filter: string;
+
+  /**  Determines if the card is opened. */
+  isOpen: boolean;
+
+  /** Called when the card is opened. */
+  onClick: (group: Beam.DirectoryEntry) => void;
```

Comment:
This should be called anytime the card's drop down button is clicked, a user can click to open or close the card.